Sources/CompressNIO/zlib/Zlib.swift

```diff
@@ -2,6 +2,13 @@
 import CCompressZlib
 import NIOCore
 
+/// Zlib algorithm
+public enum ZlibAlgorithm {
+    case gzip
+    case zlib
```

Comment:
Do you intend to potentially support more down the line?

Reply:
That is all Zlib supports